setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+meta = {}
+exec(open('meta.py').read(), meta)
+
+setup(
+    name=meta.get('NAME'),
+    version=meta.get('VERSION'),
+    description=meta.get('DESCRIPTION'),
+    long_description=''.join(open('docs/Virt-v2v-wrapper.md').readlines()),
+    keywords=meta.get('KEYWORDS'),
+    author=meta.get('AUTHOR'),
+    author_email=meta.get('EMAIL'),
+    license=meta.get('LICENSE'),
+    packages=find_packages(),
+    entry_points={
+        'console_scripts': [
+            'virt-v2v-wrapper.py = wrapper.virt_v2v_wrapper:main'
```

Comment:
Now that I checked the RPM content I see that this is wrong. Please drop the `.py` suffix. There is no need to leak the language in `/usr/bin`

Reply:
As noted in the commit message, I used this name to keep backwards compatibility for a while with consumers. If I can remove the extension I am even happier :smile: